src/pages/image/_id/report.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <div
+    class="mx-auto mt-8 mb-6 max-w-none gap-x-10 px-4 md:grid md:max-w-4xl md:grid-cols-2 md:px-6 lg:mb-30 lg:px-0 xl:max-w-4xl"
+  >
+    <figure class="mb-6 flex flex-col items-start gap-y-4">
+      <img
+        id="main-image"
+        :src="imageSrc"
+        :alt="image.title"
+        class="mx-auto h-auto w-full rounded-sm"
+        :width="imageWidth"
+        :height="imageHeight"
+      />
+      <!-- Disable reason: We control the attribution HTML generation so this is safe and will not lead to XSS attacks -->
+      <!-- eslint-disable vue/no-v-html -->
+      <caption
+        class="block w-full text-left text-sr"
+        v-html="getAttributionMarkup({ includeIcons: false })"
+      />
+      <!-- eslint-enable vue/no-v-html -->
+      <VButton
+        variant="secondary-bordered"
+        :href="`/image/${image.id}`"
+        size="disabled"
+        class="p-2 text-sr"
```

Comment:
```suggestion
        as="VLink"
        variant="secondary-bordered"
        :href="`/image/${image.id}`"
        size="disabled"
        class="p-2 text-sr text-black"
```

The additional class is required to prevent the button text from switching to the pink used for links (not sure if this is intentional or a bug in the component).